opal-gwt-client/src/main/java/org/obiba/opal/web/gwt/app/client/project/admin/ProjectAdministrationView.ui.xml

```diff
@@ -86,11 +86,14 @@
         </b:Row>
       </g:FlowPanel>
 
-      <b:Alert type="WARNING" close="false" animation="true" ui:field="noVCFServicePanel">
-        <ui:msg description="No Project VCF Store label">No VCF store is defined for this project. No genotypes can be
-          imported.
-        </ui:msg>
-      </b:Alert>
+      <b:Row>
+        <b:Column size="6">
+          <b:Alert type="WARNING" close="false" animation="true" ui:field="noVCFServicePanel">
+            <ui:msg description="No Project VCF Store label">Edit Project properties above for enabling genotype data storage.</ui:msg>
+          </b:Alert>
+        </b:Column>
+      </b:Row>
+
```

Comment:
This is not consistent with the message (content and disposition) about the database (see line 61). In addition to that we do not use this type of wording: "above", "here", etc.